lib/standards/html-elms.js

```diff
@@ -601,7 +607,8 @@ const htmlElms = {
   },
   meter: {
     contentTypes: ['phrasing', 'flow'],
-    allowedRoles: false
+    allowedRoles: false,
+    chromiumRole: 'progressbar'
```

Comment:
Should this be capitalised? If it isn't in Chromium then not, but it seems inconsistent.

Reply:
Yep, they are all inconsistent in Chrome(ium)